client/html/pages/panel/settings.html

```diff
@@ -0,0 +1,165 @@
+<template name="settings">
```

Comment:
Be careful with the indentation man, it looks like you add a 6 spaces indentation in here